features-json/css-display-contents.json

```diff
@@ -512,7 +512,7 @@
     "2":"Partial support refers to [severe implementation bug](https://hiddedevries.nl/en/blog/2018-04-21-more-accessible-markup-with-display-contents) that renders content inaccessible.",
     "3":"Buggy support. See [188259](https://bugs.webkit.org/show_bug.cgi?id=188259) and [193567](https://bugs.webkit.org/show_bug.cgi?id=193567)",
     "4":"Buttons are not accessible with `display: contents` applied. See [243486](https://bugs.webkit.org/show_bug.cgi?id=243486)",
-    "5":"HTML Tables are not accessible with `display: contents` applied. See [239479](https://bugs.webkit.org/show_bug.cgi?id=239479) and [239478](https://bugs.webkit.org/show_bug.cgi?id=239478)"
+    "5":"HTML Tables are not accessible with `display: contents` applied. See issues for [Chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=1366037), [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1791648), and [WebKit](https://bugs.webkit.org/show_bug.cgi?id=243486)"
```

Comment:
@jensimmons Looks like you updated the section for Tables (5) here when you meant to update the Buttons one (4)?